.gitignore

```diff
@@ -60,9 +60,11 @@
 /build/msw/vc*_mswud
 /build/msw/wx_vc8arm_*.vcproj
 /build/msw/wx_vc8arm.sln
-/build/msw/ipch
-/build/msw/wx_vc12.opensdf
-/build/msw/wx_vc12.sdf
+/build/msw/ipch/
+/build/msw/*.opensdf
+/build/msw/*.sdf
+/build/msw/*.VC.db
+/build/msw/*.VC.VC.opendb
```

Comment:
This one looks a bit strange, does MSVS 2017 really use `.VC.VC` in its files? 2015 doesn't seem to...